dace/sdfg.py

```diff
@@ -220,13 +220,10 @@ def __init__(self,
         if name is not None and not validate_name(name):
             raise InvalidSDFGError('Invalid SDFG name "%s"' % name, self, None)
 
-        #if not isinstance(arg_types, collections.OrderedDict):
-        #    raise TypeError
-
-        #self._arg_types = arg_types  # OrderedDict(str, typeclass)
-        #self._constants = constants  # type: Dict[str, Any]
         self.arg_types = arg_types or collections.OrderedDict()
-        self.constants_prop = constants or {}
+        self.constants_prop = {}
```

Comment:
Why is it that we need a separate `constants` and `constants_prop`? Seems pretty ugly

Reply:
Having only one structure would require many more changes in the code. Feel free to amend, no particular reason why it is this way.

Reply:
Ok, not worth it now